app/assets/src/components/PipelineSampleReads.jsx

```diff
@@ -917,11 +872,20 @@ class PipelineSampleReads extends React.Component {
                   </a>
                   <span className={cs.rightArrow}>{">"}</span>
                 </div>
```

Comment:
This one should be the same component as the pre-title Phylogenetic Tree below..